build.sbt

```diff
@@ -6,3 +6,7 @@ lazy val contrib = project.
 
 lazy val mqtt = project.
   enablePlugins(AutomateHeaderPlugin)
+
+lazy val amqp = project.
+  enablePlugins(AutomateHeaderPlugin)
+  .configs(IntegrationTest)
```

Comment:
Was not sure as well. After a bit of poking around, it looks like we can set it in `amqp/build.sbt` by adding:

```scala
lazy val amqp = (project in file(".")).
  configs(IntegrationTest)
  enablePlugins(AutomateHeaderPlugin)
```

Enabling plugins work there as well. So we can leave `~/build.sbt` with only

```scala
lazy val amqp = project
```